eslint/babel-eslint-parser/src/configuration.cjs

```diff
@@ -11,7 +11,7 @@ exports.normalizeESLintConfig = function (options) {
 
   return {
     babelOptions: { cwd: process.cwd(), ...babelOptions },
-    ecmaVersion,
+    ecmaVersion: ecmaVersion === 'latest' ? 2020 : ecmaVersion,
```

Comment:
Does it throw if we just directly pass `"latest"`?

Reply:
I don't know, I can't find where it's used.

Reply:
But yes, if we use `"latest"`, the parser throws https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1465/checks?check_run_id=3248815028#step:5:18825

Reply:
`eslint-scope` does not support `'latest'` as valid. Does is make more sense to update it there?

Reply:
`eslint-scope` suggest handling on parser side. https://github.com/eslint/eslint-scope/issues/74#issuecomment-898780125